playerbot/strategy/triggers/HealthTriggers.h

```diff
@@ -29,14 +29,23 @@ namespace ai
 	class HealthInRangeTrigger : public ValueInRangeTrigger
 	{
 	public:
-		HealthInRangeTrigger(PlayerbotAI* ai, string name, float maxValue, float minValue = 0) : ValueInRangeTrigger(ai, name, maxValue, minValue) {}
+		HealthInRangeTrigger(PlayerbotAI* ai, string name, float maxValue, float minValue = 0, bool isTankRequired = false) : ValueInRangeTrigger(ai, name, maxValue, minValue) 
+        {
+            this->isTankRequired = isTankRequired;
+        }
 
 		virtual bool IsActive()
 		{
-		    return ValueInRangeTrigger::IsActive() && !AI_VALUE2(bool, "dead", GetTargetName());
+		    return ValueInRangeTrigger::IsActive() 
+                && !AI_VALUE2(bool, "dead", GetTargetName())
+                && (!isTankRequired || 
+                    (GetTarget()->IsPlayer() && ai->IsTank((Player*)GetTarget(), false)));
 		}
 
 		virtual float GetValue();
+
+    protected:
+        bool isTankRequired;

```

Comment:
This code is not well tabulated

Reply:
There is 
![obraz](https://user-images.githubusercontent.com/28183654/236527668-e9b5dad6-afff-4934-aafd-91f1ebcd46f7.png)

Github messing up.